gserver/src/steps.handler.ts

```diff
@@ -359,7 +358,12 @@ export default class StepsHandler {
             //We can replace some outputs, ex. strings in brackets to make insert strings more neat
             res = res.replace(/\"\[\^\"\]\+\"/g, '""');
```

Comment:
It's fine to not have this option. My concerne was to not change current habits of users if they want to

Reply:
Is there some use case for the following scenarios:

<img width="696" alt="image" src="https://github.com/alexkrechik/VSCucumberAutoComplete/assets/138122895/3dc47cfd-ff81-4f75-bab8-a83b98a616fd">

This always seemed like a bug to me rather than desirable functionality.

Happy to discuss this further as, like you say, we don't want to change current user habits if they are actually benefitting from the current implementation.

Reply:
> This always seemed like a bug to me rather than desirable functionality.

same for me